playbooks/bosh-acceptance-tests/run.yaml

```diff
@@ -0,0 +1,137 @@
+
+          bosh alias-env bosh-1 -e ${bosh_ip} --ca-cert <(bosh int ./creds.yml --path /director_ssl/ca)
+          export BOSH_CLIENT=admin
+          export BOSH_CLIENT_SECRET=`bosh int ./creds.yml --path /admin_password`
+          bosh -e bosh-1 env
+
+          # path to the stemcell you want to use for testing
+          export BAT_STEMCELL="{{ ansible_user_dir }}/bosh-1/bosh-openstack-kvm-ubuntu-trusty-go_agent.tgz"
+
+          # path to the bat yaml file which is used to generate the deployment manifest (see below `bat.yml`)
+          export BAT_DEPLOYMENT_SPEC="{{ ansible_user_dir }}/bosh-1/bat.yml"
+
+          # BOSH CLI executable path
+          export BAT_BOSH_CLI=bosh
+
+          # DNS host or IP where BOSH-controlled PowerDNS server is running, which is required for the DNS tests. For example, if BAT is being run against a MicroBOSH then this value will be the same as BAT_DIRECTOR
+          export BAT_DNS_HOST=8.8.8.8
+
+          # the name of infrastructure that is used by bosh deployment. Examples: aws, vsphere, openstack, warden, oci.
+          export BAT_INFRASTRUCTURE=openstack
+
+          # the type of networking being used: `dynamic` or `manual`.
+          export BAT_NETWORKING=dynamic
+
+          # the path to ssh key, used by OS specs to ssh into BOSH VMs
+          export BAT_PRIVATE_KEY="{{ ansible_user_dir }}/bosh-1/microbosh.pem"
+
+          # Run tests with --fail-fast and skip cleanup in case of failure (optional)
+          export BAT_DEBUG_MODE=true
+
+          export BOSH_OS_BATS=true
+
+          export BOSH_ENVIRONMENT=bosh-1
+          # export BOSH_CLIENT=<director username>
+          # export BOSH_CLIENT_SECRET=<director password>
+          # export BOSH_CA_CERT=<director ca cert content or path>
+          cat << EOF >> bat.yml
+          ---
+          cpi: openstack
+          properties:
+            stemcell:
+              name: bosh-openstack-kvm-ubuntu-trusty-go_agent
+              version: latest
+            pool_size: 1
+            instances: 1
+            instance_type: ${SMALL_FLAVOR}
+            availability_zone: ${OS_AVAILABILITY_ZONE}
+            flavor_with_no_ephemeral_disk: ${MEDIUM_FLAVOR}
+            vip: 0.0.0.43 # Virtual (public/floating) IP assigned to the bat-release job vm, for ssh testing
+            networks:
+            - name: default
+              type: dynamic
+              cloud_properties:
+                net_id: ${NET_ID} # Network ID
+                security_groups: [bosh-${BUILD_ID}] # security groups assigned to deployed VMs
+            key_name: microbosh-${BUILD_ID} # (optional) SSH keypair name
+          EOF
+
+          pushd '{{ ansible_user_dir }}/{{ zuul.project.src_dir }}'
+          gem install bundler
+          bundle install
+          cp '{{ ansible_user_dir }}/bosh' /usr/local/bin/bosh
+          bundle exec rspec spec --tag ~vip_networking --tag ~dynamic_networking --tag ~root_partition --tag ~raw_ephemeral_storage
```

Comment:
why we do not run vip_networking and dynamic_networking test cases?

Reply:
just follow the example of the bosh-acceptance-tests, may we should enable all the tests firstly, and to see what tests should be skipped in next step, will change this.